.github/pull_request_template.md

```diff
@@ -13,12 +13,12 @@ This save file can be used to play through the new mission content:
 
 ## Artwork Checklist
  - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
- - [ ] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}
+ - [ ] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}
  - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}
 
 
 -----------------------
-**Bugfix:** This PR addresses issue #{{insert number}}
+**Bugfix:** This PR fixes #{{insert number}}
```

Comment:
I intentionally did not use "fixes" in the template, because while a contributor may think their PR addresses the issue completely, that is not their call to make.

Reply:
Oh I see, thanks!